site/v5/features/features-multitenant.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Multitenant 
+date: 2014-04-18 11:30:00 
+permalink: /v5/features/multitenant/index.html
+toc: true
+eleventyNavigation:
+  version: v5
+  order: 15
+  parent: features
+  key: features multitenant 
+  title: Multitenant
+---
+<h1 class="title">Multitenant <span class="professional"><a href="/pro/index.html">PRO</a></span></h1>
+
+
+
+[[TOC]]
+## Introduction 
+This feature helps in situations when it's required to setup multiple tenants in a single project. A common scenario is a SaaS product in which, instead of having a shared database for all the clients, each has its own and independent database(in the same or different server), but sharing the same source code and deployment. In this case, you would want to apply the same Mongock migration to each client's database. 
+<p class="text-center">
+    <img src="/images/multitenant-overview.png" alt="multitenant overview" width="75%">
+</p> 
+
+## How it works
+The application is configured to work with N datasources and Mongock runs the migration for all of them(sequentially). To achieve this, the user has to configure a Mongock driver per datasource, pass them to the Mongock builder and execute the Mongock runner which will apply the migration for all the tenants.
+
+
+## With Mongock Standalone
+This is the simplest scenario as there is no ORM framework to deal with. It just requires passing a driver per tenant to the Mongock builder. This mechanism is standard and doesn't change among the different Mongock drivers/databases
+```java
+  MongockStandalone.builder()
+    .setDriverMultiTenant(
+      MongoSync4Driver.withDefaultLock(mongoClientTenant1, "db-tenant-1"),
+      MongoSync4Driver.withDefaultLock(mongoClientTenant2, "db-tenant-2")
+    )
+  //...
+```
+
+## With Springdata
+While the core idea and how it works still remains, when using Springdata in a multitenant environment(independently of Mongock) it requires some extra work, which it's explained bellow.
+
+### The problem
+Springdata hides all the complexity to connect to the database, which helps a lot, but it also means the developer needs to give some flexibility up . In this case, he cannot easily specify which tenant is connecting to. 
```

Comment:
Why is the developer a 'he'? :) can we rephrase the last sentence please?

Reply:
In this case, it's not straightforward specifying the database which we want to connect to.